earth_enterprise/src/common/khCache.h

```diff
@@ -22,6 +22,10 @@
 #include <vector>
 
 #include "khTypes.h"
+#include "fusion/autoingest/MiscConfig.h"
```

Comment:
I don't think you need the MiscConfig, khRefCounter, or SharedString includes any more.